wcivf/apps/elections/helpers.py

```diff
@@ -81,19 +81,23 @@ def get_data(self, election_id):
         self.ee_cache[election_id] = None
         return None
 
-    def iter_recently_modified_election_ids(self):
+    def iter_recently_modified_election_ids(self, group_type=None):
+        modified_date = timezone.datetime.now().date() - timezone.timedelta(days=3)
+
         params = {
-            "modified": timezone.datetime.now().date()
-            - timezone.timedelta(hours=1),
+            "modified": modified_date,
```

Comment:
Is this change from t-1hr to 1-3days intentional?

Reply:
Given the cron, I think this should be moved back to 1 hour, but I was adapting this as needed for testing purposes. I should not have committed the change.